src/app/components/student-form/student-form.component.ts

```diff
@@ -5,24 +5,34 @@ import { ActivatedRoute, Router, NavigationEnd } from '@angular/router';
 import { Student } from 'src/app/models/student.model';
 import { filter } from 'rxjs/operators';
 import { MessagesService } from '../../services/messages.service';
+import nations from '../../countries.json';
 
 @Component({
   selector: 'app-student-form',
   templateUrl: './student-form.component.html',
   styleUrls: ['./student-form.component.scss']
 })
 export class StudentFormComponent implements OnInit {
+  bsValue = new Date();
+  bsRangeValue: Date[];
+  maxDate = new Date();
+
   public isEdit: boolean;
-  countries = ['USA', 'Canada', 'Uk', 'Australia', 'Costa Rica'];
+
+  public countries = nations.countries;
```

Comment:
what is the difference between nations and countries in this case? why this var is named `nations` ?